client/my-sites/plans/jetpack-plans/product-lightbox/payment-plan.tsx

```diff
@@ -67,6 +68,8 @@ const PaymentPlan: React.FC< PaymentPlanProps > = ( {
 	const currentPrice = isNumber( discountedPrice ) ? discountedPrice : originalPrice;
 	const currencyCode = useSelector( getCurrentUserCurrencyCode ) || 'USD';
 
+	const tooltipText = productTooltip( product, priceTierList, currencyCode ?? 'USD' );
```

Comment:
Nitpick: I'm not really a fan of this `productTooltip(...)` function.. The reasons being because how tightly coupled it is to it's arguments, and the returned translated text being coupled with all the contents (including all the outer html).
I think ideally this could be a custom hook, i.e.- `useGetSearchTooltipContent( item: SelectorProduct );` so that it could be called anywhere you have access to the SelectorProduct (item). And the returned content could be a more extensible/flexible object in the form of:
```
{
title: TranslateResult | ReactNode,
text: TranslateResult | ReactNode,
}
```
Something like that...

But I understand that the `productTooltip(...)` function already existed and is being used prior to this PR, so not a blocker, just thought I would mention it for possible future improvement.

Reply:
Thanks for the feedback. I completely agree with you, but I'd say we can refactor that method in another PR, so I'll go ahead and ship this one.